tests/rkt_auth_test.go

```diff
@@ -50,8 +49,8 @@ type genericAuthTest struct {
 func TestAuthBasic(t *testing.T) {
 	tests := []genericAuthTest{
 		{"basic-no-config", false, "", authFailedDownload},
-		{"basic-local-config", true, common.DefaultLocalConfigDir, authSuccessfulDownload},
-		{"basic-system-config", true, common.DefaultSystemConfigDir, authSuccessfulDownload},
+		{"basic-local-config", true, "local", authSuccessfulDownload},
+		{"basic-system-config", true, "system", authSuccessfulDownload},
```

Comment:
Could "local" and "systemd" be an enum instead of a string? By enum in golang, I mean some constants defined with the iota pattern.

Or do some tests really need a string?